Going through training manual: Fixing incorrect inline tags (Part 5) (#10393)
qgis/QGIS-Documentation: docs/training_manual/processing/cutting_merging.rst

of the city area, and then we buffer it, so as to have a raster layer that
covers a bit more that the strictly necessary.

To calculate the bounding box , we can use the :guilabel:`Polygon from layer extent` algorithm

.. figure:: img/cutting_merging/bbox.png

> To buffer it, we use the :guilabel:`Fixed distance buffer` algorithm, with the following parameter values.

Reviewer: Here also
```suggestion
To buffer it, we use the :guilabel:`Buffer` algorithm, with the following parameter values.
```

.. figure:: img/cutting_merging/buffer.png

It is a rounded box, but we can easily get the equivalent box with square angles,
by running the :guilabel:`Polygon from layer extent` algorithm on it. We could have buffered
the city limits first, and then calculate the extent rectangle, saving one step.

.. figure:: img/cutting_merging/buffer_squared.png

You will notice that the rasters has a different projection from the vector.
We should therefore reproject them before proceeding further, using the
:guilabel:`Warp (reproject)` tool.

.. figure:: img/cutting_merging/warp.png

.. note:: Recent versions have a more complex interface. Make sure at least
 one compression method is selected.

With this layer that contains the bounding box of the raster layer that we want

> to obtain, we can crop both of the raster layers, using the :guilabel:`Clip raster with polygon` algorithm.

Reviewer: ```suggestion
to obtain, we can crop both of the raster layers, using the :guilabel:`Clip raster by mask layer` algorithm.
```

.. figure:: img/cutting_merging/slopereproj.png

The slope produced by the :guilabel:`Slope, Aspect, Curvature` algorithm can be expressed
in degrees or radians; degrees are a more practical and common unit.

> In case you calculated it in radians, the :guilabel:`Metric conversions` algorithm will

Reviewer: I didn't know and couldn't find this alg in default providers. So probably should we consider later to switch to raster calculator, here.

help us to do the conversion (but in case you didn't know that algorithm existed,
you could use the raster calculator that we have already used).

.. figure:: img/cutting_merging/metricconversions.png

Reprojecting the converted slope layer back with the :guilabel:`Reproject raster layer`,
we get the final layer we wanted.

> .. warning:: todo: Add image

Reviewer: I don't know if this adds any (positive) value, here
```suggestion
```
